.gitignore

```diff
@@ -2,7 +2,7 @@ __pycache__
 *.egg-info
 build
 .DS_STORE
-comfyui/
+./comfyui/
 ComfyUI/
 cuda_temp
```

Comment:
In `.gitignore` isn't this the same as `comfyui/`. If you want to only ignore the top folder you should do `/comfyui` I think.

Reply:
> In `.gitignore` isn't this the same as `comfyui/`. If you want to only ignore the top folder you should do `/comfyui` I think.

The ./ prefix makes it path-specific, referring to the comfyui directory only in the root of the repository. Otherwise this will match on `workflows/comfyui` and prevent changes to the workflows from being tracked

Reply:
Not a big deal but as far as I remember, `.gitignore` patterns are always implicitly relative (see [[Git documentation](https://git-scm.com/docs/gitignore)](https://git-scm.com/docs/gitignore)). You can test this behavior by creating a new file. I believe you meant `/comfyui`. Also, why do we want to ignore `comfyui` only at the root level but `ComfUI` in all folders?

Reply:
You're right, updated